storage/perfschema/pfs_digest.cc

```diff
@@ -47,7 +47,7 @@ bool flag_statements_digest= true;
   Current index in Stat array where new record is to be inserted.
   index 0 is reserved for "all else" case when entire array is full.
 */
-volatile uint32 digest_index;
+PFS_ALIGNED volatile uint32 digest_index;
```

Comment:
probably. The perf schema component is a rather a minimal poor port of the MySQL-5.7 changes